packages/gravity-vue/src/components/icon/icon.vue

```diff
@@ -0,0 +1,109 @@
+<script lang="ts">
+import type { QAProps } from '../../types'
+import type { Component, VNode } from 'vue'
+import { computed, ref } from 'vue'
+import { block } from '../../utils/cn'
+import type { SVGIconSpriteData } from './types'
+
+export type IconData = string | Component | ((ctx: any, cache: any) => VNode)
```

Comment:
этого будет достаточно
```suggestion
export type IconData = string | Component
```